src/isar_exr/api/energy_robotics_api.py

```diff
@@ -20,56 +22,86 @@
     PointOfInterestTypeEnum,
     PointOfInterestActionPhotoInput,
 )
+from gql.dsl import DSLVariableDefinitions, DSLMutation, DSLQuery, dsl_gql
+
+
+def to_dict(obj):
+    return json.loads(json.dumps(obj, default=lambda o: o.__dict__))
 
 
 class EnergyRoboticsApi:
     def __init__(self):
         self.client = GraphqlClient()
```

Comment:
Would consider adding 
`self.schema = self.client.schema`
for easier accessibility of schema